README.md

```diff
@@ -60,7 +60,7 @@ $ pipenv run python run_tests.py
 
 Run linting (the travis build sets a custom max line length)
 ```
-$ pipenv check --style . --max-line-length 100
+$ pipenv check --style . --max-line-length 120
```

Comment:
Is 120 the line length we have decided to use?

Reply:
Yes because if you run the original command in the readme you'll get a million errors 😆